<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="txt_user">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>test@test.de</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_password">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
   </property>
   <property name="text">
    <string>test</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QFrame" name="Info_Frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>0</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="txt_Info_Frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>371</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Information</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_OK">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>240</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Cancel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_username">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>E-Mail:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_password">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Password: </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
